### PR TITLE
Provider token refresh

### PR DESCRIPTION
**Pull request by talaharoni**
## Related Issues
Related to https://github.com/descope/etc/issues/6852

### CODE REVIEW

File: lib/management/user.ts [hunk 1]
```diff
@@ -521,15 +521,24 @@ const withUser = (sdk: CoreSdk, managementKey?: string) => {
...
      * @param provider the provider name (google, facebook, etc.).
+     * @param withRefreshToken optional, include the refresh token in the response
+     * @param forceRefresh optional, force to refresh the token
      * @returns The ProviderTokenResponse of the given user and provider
      */
     getProviderToken: (
       loginId: string,
       provider: string,
+      withRefreshToken?: boolean,
+      forceRefresh?: boolean,
```
Comment (itaihanski) [inline lib/management/user.ts:532]: I'd consider wrapping these in an "options object" so we can add future options without breakage

File: lib/management/user.ts [hunk 2]
```diff
@@ -521,15 +522,23 @@ const withUser = (sdk: CoreSdk, managementKey?: string) => {
      * Note: The 'Manage tokens from provider' setting must be enabled.
      * @param loginId the login ID of the user
      * @param provider the provider name (google, facebook, etc.).
+     * @param withRefreshToken optional, include the refresh token in the response
+     * @param forceRefresh optional, force to refresh the token
```
Comment (itaihanski) [inline lib/management/user.ts:526]: Need to update this to reflect your change

**Review by itaihanski — APPROVED** [review]

File: lib/management/user.ts (hunk 1 above)
Comment (talaharoni) [reply lib/management/user.ts]: done

File: lib/management/user.ts (hunk 2 above)
Comment (talaharoni) [reply lib/management/user.ts]: updated

**Review by itaihanski — APPROVED** [review]

**Pull request merged**